src/components/Table/TableStyles.tsx

```diff
@@ -218,17 +219,6 @@ export const cardStyle: GridStyle = {
   },
 };
 
-/** GridTable as Table utility to apply <tr> element override styles. */
-export function tableRowStyles(as: RenderAs, column?: GridColumn<any>) {
```

Comment:
This moved from being called in _every_ cell to now being applied in the `cellCss` defined in `Row.tsx`.